src/Client/Invoice.php

```diff
@@ -85,6 +85,41 @@ public function getInvoice(
         }
     }
 
+    public function getAllInvoices(string $storeId): \BTCPayServer\Result\InvoiceList
+    {
+        return $this->_getAllInvoicesWithFilter($storeId, null);
+    }
+
+    public function getInvoicesByOrderId(string $storeId, string $orderId): \BTCPayServer\Result\InvoiceList
```

Comment:
This seems very opinionated and not an actually dedicated greenfield api but just a variation.

Reply:
I could not get the orderId as array to work and opened a bug report for it. In the meantime, I've gotten news and can fix the PR now so it will work. The notation is weird, but appears to be framework-specific. See https://github.com/btcpayserver/btcpayserver/issues/2736

Reply:
I created the private method to avoid copy/pasting the same code twice, so it's not useless. The naming could be better perhaps. I just needed to name it something and this made sense to me.
At present there are no other filter options (yet). I can alway pick a different name.

Reply:
@Kukks we set out to build an opinionated library that is as easy to use as we can imagine.
At the same time, we're not limiting the developer, so it's not like the dev is at a disadvantage.

Reply:
ah year, I also new only the [] pattern and the comma separated one that is kinda JSON api related, good to know

Reply:
I fixed this method just now.